pom.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
According task pull request must be to develop and at master should be default project